src/Assignment4/Header/index.js

```diff
@@ -0,0 +1,15 @@
+import "./styles.css";
+import logo from "../../logo/React.webp";
+import usericon from "../../logo/usericon.png";
+import searchicon from "../../logo/searchicon.png";
+
+const Header = ({}) => (
```

Comment:
You can remove the empty curly braces

Reply:
this empty curly braces is fine. its like the developer is ready for any future de structuring. doesnt have any impact.